index.html

```diff
@@ -0,0 +1,159 @@
+                <p>Create</p>
+               </div>
+        </span>
+        <span class="i1">
+            <ion-icon class="icons" name="apps-outline"></ion-icon>
+            <div class="text1">
+                <p>YouTube Apps</p>
+               </div>
+        </span>   
+        <span class="i1">
+            <ion-icon class="icons" name="notifications-outline"></ion-icon>
+            <div class="text1">
+                <p>Notifications</p>
+               </div>
+        </span> 
+        <span class="i1">
+            <ion-icon class="icons" name="person-circle-outline"></ion-icon>
+        </span> 
+    </div>
+    </nav>
+    <div class="sidebar">
+        <ion-icon name="menu-outline" class="menu"></ion-icon>
+     <ul>
+         <li>
+             <a href="#">
+                 <span><ion-icon name="home"></ion-icon></span>
+                 <span>
+                     <p>Home</p>
+                 </span>
+             </a>
+         </li>
+         <li>
+            <a href="#">
+                <span><ion-icon name="compass-outline"></ion-icon></span>
+                <span>
+                    <p>Explore</p>
+                </span>
+            </a>
+        </li>
+        <li>
+            <a href="#">
+                <span><ion-icon name="flash-outline"></ion-icon></span>
+                <span>
+                    <p>Shorts</p>
+                </span>
+            </a>
+        </li>
+        <li>
+            <a href="#">
+                <span><ion-icon name="film-outline"></ion-icon></span>
+                <span>
+                    <p>Subscriptions</p>
+                </span>
+            </a>
+        </li>
+        <li>
+            <a href="#">
+                <span><ion-icon name="file-tray-outline"></ion-icon></span>
+                <span>
+                    <p>Library</p>
+                </span>
+            </a>
+        </li>
+     </ul>
+    </div>
+    <hr>
+    <div class="main">
+        <div class="collums">
+          <ul>
+              <li class="all">
+                  <a href="#">All</a>
+              </li>
+              <li>
+                <a href="#">css</a>
+            </li>
+            <li>
+                <a href="#">Scene</a>
+            </li>
+            <li>
+                <a href="#">Thrillers</a>
+            </li>
+            <li>
+                <a href="#">Human</a>
+            </li>
+            <li>
+                <a href="#">Live</a>
+            </li>
+            <li>
+                <a href="#">Reacts</a>
+            </li>
+            <li>
+                <a href="#">Gaming</a>
+            </li>
+            <li>
+                <a href="#">Sports</a>
+            </li>
+            <li>
+                <a href="#">Php</a>
+            </li>
+            <li>
+                <a href="#">Laravel</a>
+            </li>
+            <li>
+                <a href="#">Movies</a>
+            </li>
+          </ul>
+        </div>
```

Comment:
Please use HTML5 Semantic tags in your codes, for example, header, nav, main over divs

Reply:
Please use HTML5 Semantic tags in your codes, for example, header, nav, main over divs this will  provide additional information about that document, which aids in communication

Reply:
- Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change.